Gruntfile.js

```diff
@@ -317,7 +317,7 @@ module.exports = function( grunt ) {
 
 			// Add className, height, and width to the svg element
 			fileContent = fileContent.slice( 0, 4 ) +
-						' className={ iconClass } height={ size } width={ size } onClick={ onClick }' +
+						' className={ iconClass } height={ size } width={ size } onClick={ onClick } { ...otherProps }' +
```

Comment:
@adambbecker looks like we can override classname, so we'll want to add special handling for that prop

cc @folletto